tests/test_helpers.py

```diff
@@ -32,6 +32,27 @@
       "--authors", "sbillinge", "ashaaban", "--abstract", "the earth is round as seen from mars",
       "--title", "On the roundness of the Earth", "--status", "in-prep"],
      "2006as_mars has been added in presentations\n"),
+    (["helper", "l_progress", "-l", "ascopatz"],
+     "*************************[Proposed Projecta]*************************\n"
+      "20sb_firstprojectum\n"
```

Comment:
I think I would like to know what the projectum is even when it is not verbose.  Maybe add the description above the status and dates?

Reply:
The description (line 44) is printed under the status and the date (line 43). So I just switched the order of these two lines.